README.md

```diff
@@ -31,22 +31,67 @@ Once installed, the following ``py.test`` command-line parameters are available:
 
 ```bash
 py.test \
-    [--ansible-inventory <path_to_inventory>] \
-    [--ansible-host-pattern <host-pattern>] \
-    [--ansible-connection <plugin>] \
-    [--ansible-module-path <path_to_modules] \
-    [--ansible-user <username>] \
-    [--ansible-sudo] \
-    [--ansible-sudo-user <username>]
+    [--inventory <path_to_inventory>] \
+    [--host-pattern <host-pattern>] \
+    [--connection <plugin>] \
+    [--module-path <path_to_modules] \
+    [--user <username>] \
+    [--become] \
```

Comment:
This suggests that pytest-ansible will only be compatible with ansible >= 1.9. Didn't see this mentioned in the README - seems like it would be helpful to note this somewhere.